projects/gcsweb/pipeline.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: v1
+kind: Template
+objects:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    name: gcsweb
+- kind: BuildConfig
+  apiVersion: v1
+  metadata:
+    name: gcsweb-binary
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: gcsweb:binaries
+    source:
+      git:
+        uri: https://github.com/openshift/test-infra.git
+      contextDir: gcsweb
+      type: Git
+      dockerfile: |
+        FROM openshift/origin-release:golang-1.7
+
+        ENV ARCH=amd64 VERSION=v1.0.4 PKG=k8s.io/test-infra/gcsweb
+        COPY . /go/src/k8s.io/test-infra/gcsweb
+        RUN cd /go/src/k8s.io/test-infra/gcsweb && \
+            ./build/build.sh
+        RUN mkdir -p /go/src/k8s.io/test-infra/gcsweb/bin/${ARCH}/
+        RUN mv /go/bin/gcsweb /go/src/k8s.io/test-infra/gcsweb/bin/${ARCH}/
+    strategy:
+      dockerStrategy:
```

Comment:
@smarterclayton this is not allowed in the CI cluster. Do we support golang STI builds? Otherwise, I am not sure how to compile prow using Builds/ImageStreams ...